packages/flutter/lib/src/services/url_launcher.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:sky_services/flutter/platform/url_launcher.mojom.dart' as mojo;
+import 'shell.dart';
+
+mojo.UrlLauncherProxy _initURLLauncherProxy() {
```

Comment:
s/_initURLLauncherProxy/_initUrlLauncherProxy/

(These are from the Dart style guide.)